src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -272,7 +273,11 @@ public void downloadZippedFile(Environment env, HttpServletRequest request,
 						folderNameMap = Utils.updateFolderNameMap(
 								folderNameMap, title, folderName);
 						if (folderNameMap.containsKey(folderName)) {
-							folderName = folderNameMap.get(folderName);
+							// if the folder name have / in it then we are not zipping the file with original name instead the folder
+							// name will contain _ in it. As having the / will have cause the zip library creating inner folders
+							if (!(StringUtils.countOccurrencesOf(folderNameMap.get(folderName), "/") > 1)) {
```

Comment:
Did not see the String replacement of "/" to "_" when the folder name contains "/"